Translations/Localization.md

```diff
@@ -0,0 +1,92 @@
+> Localization describes the subsequent process of translating an internationalized plugin. Localization is often abbreviated as l10n (because there are 10 letters between the l and the n.)
+
+This is a handy up-to-date how-to for generating WordPress Theme/Plugin translation files with WP-CLI. The following workflow is covered: creating a POT (Portable Object Template) file, copying it to a PO (Portable Object) file where the `msgstr` sections are translated or updating pre-existing PO file and finally building the MO (Machine Object) file which is used by `gettext` functions to output translations to the user.
+
+### Available commands:
+
+- [wp i18n make-pot](https://github.com/wp-cli/i18n-command#wp-i18n-make-pot)
+- [wp i18n make-json](https://github.com/wp-cli/i18n-command#wp-i18n-make-json)
+- [wp i18n make-mo](https://github.com/wp-cli/i18n-command#wp-i18n-make-mo)
+- [wp i18n update-po](https://github.com/wp-cli/i18n-command#wp-i18n-update-po) - ^2.4.0 WP CLI i18n-command [package](https://github.com/wp-cli/i18n-command) is required.
+
+### Useful Notes
+- The	following files and folders are always excluded: `node_modules, .git, .svn, .CVS, .hg, vendor, Gruntfile.js, webpack.config.js, *.min.js`.
+- By default, the "Text Domain" header of the plugin or theme is used. If the header is not specified you'll need to add an argument to the CLI command with text-domain it should search for: `[--domain=<domain>]`.
+- when naming translation files use the [WP Locales](https://make.wordpress.org/polyglots/teams/) naming convention. Example for creating files for French language translation: fr_FR.pot, fr_FR.po, fr_FR.mo.
+- `[--debug]` global WP CLI option, outputs info during runtime.
+- Xdebug PHP extension with `develop` mode feature turned on can cause issues when running WP CLI I18n commands on the entire theme or plugin folder. Easiest way to get around this is to turn it off for the current session by prepending `XDEBUG_MODE=debug` to the command: `XDEBUG_MODE=debug wp i18n make-po <source> [<destination>]`.
+- The POT and PO files can easily be interchangeably renamed to change file types without issues.
+
+
+Most of the WP CLI i18n-command defaults are fine but you might want to add these flags:
```

Comment:
```suggestion

Most of the WP CLI i18n-command defaults are fine but you may want to add these flags:
```

Even though may/might have similar meanings, and might describes the lesser certainty, the 'may' reads better in this case (for me at least).

Reply:
Now that you brought that up I'm not sure myself either but according to [this](https://www.masterclass.com/articles/may-vs-might-explained) and some other resources "might" is more suggestive and "may" is more permissive. 🤷🏼 Willing to go either way 😄

Reply:
@infinum/wordpress-team what reads better to you might or may? may: 👍🏼 might: 🚀

Reply:
Having added the part about creating JED files to localize JS files, this note is now obsolete. We wouldn't want to exclude JS files nor the `public/` folder anymore because the main JED file is created from the `public/applicationBlocksEditor.js` file.

Reply:
Ok, just to confirm, you were right. Might is more common, may is more polite.